feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.spec.ts

```diff
@@ -109,9 +109,11 @@ describe('ImportEntriesDialogComponent', () => {
     it('should call loadProductsToCart method', () => {
       loadProducts$.next(mockLoadProduct);
       component.importProducts({
-        name: mockName,
-        description: '',
         products: mockProducts,
+        savedCartInfo: {
+          name: mockName,
+          description: '',
```

Comment:
Worth to consider is to set `description` param as optional.

```
    savedCartInfo?: {
      name: string;
      description?: string;
    };
```

Reply:
the `description` is defined (it's empty string if not filled field in form) in every case we use `savedCartInfo`